include/swift/ClangImporter/ClangImporter.h

```diff
@@ -773,6 +773,11 @@ getPrivateFileIDAttrs(const clang::CXXRecordDecl *decl);
 ///
 /// Returns false if \a decl was not imported by ClangImporter.
 bool declIsCxxOnly(const Decl *decl);
+
+/// Construct the internal Swift name for an imported Clang operator kind.
+/// \a op must represent an actual operator, i.e., not Clang::OO_None or
+/// Clang::NUM_OVERLOADED_OPERATORS.
+Identifier getOperatorName(ASTContext &ctx, clang::OverloadedOperatorKind op);
```

Comment:
If we only want to use this within the importer, I think `ImporterImpl.h` is a better place to put this. Things in this header are public to other parts of the compiler.

Reply:
That makes sense! In fact, now that I look, there is already a very similar function in `ImporterImpl.h`. It would also be nice to have these helpers in the same place.

Reply:
Just a heads up: I removed that function (`getPrivateOperatorName()`) and rewrote it as an overload of `getOperatorName()` that doesn't allocate `std::string`s unnecessarily.